runtime/lookup.c

```diff
@@ -625,7 +625,7 @@ lookupFindTable(uchar *name)
 {
 	lookup_ref_t *curr;
 
-	for(curr = loadConf->lu_tabs.root ; curr != NULL ; curr = curr->next) {
+	for(curr = loadConf->lu_tabs.last ; curr != NULL ; curr = curr->next) {
```

Comment:
It solved a problem with the configuration file in an earlier version, but I just noticed the lookupNew function has changed. So you are right, this change is not needed.